### PR TITLE
Release 3.2.1

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

- https://github.com/mattermost/mattermost-plugin-jira/pull/868

#### Ticket Link

https://github.com/mattermost/mattermost-plugin-jira/issues/876

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -65,6 +65,7 @@ endif
 ## Ensures NPM dependencies are installed without having to run this all the time.
 webapp/.npminstall:
 ifneq ($(HAS_WEBAPP),)
+	git config --global url."ssh://git@".insteadOf git://
```
Comment (hanzei) [inline Makefile:68]: Do we need that or can we wait for the orb release?

Comment (hanzei) [reply Makefile:68]: Let's remove this

**Review by hanzei — CHANGES_REQUESTED** [review]
See comment

**Review by hanzei — APPROVED** [review]

**Comment by mickmister** [comment]
Superseded by https://github.com/mattermost/mattermost-plugin-jira/pull/884

**Pull request closed without merging**